rviz2/scripts/rviz1_to_rviz2.py

```diff
@@ -0,0 +1,489 @@
+#!/usr/bin/env python3
```

Comment:
Per Jacob's request, is it possible to add a simple unittest?
Perhaps not full coverage, but converts a real ROS1 rviz file to ROS2 rviz2 without error?

(ideally, there'd be a way for rviz to validate a config file w/o display or perhaps under `xvfb`; however, not sure practical that is)

Reply:
I added a couple checks to make sure the script runs and produces valid yaml output in 406426a28a4d93e879f25f0e379a936c1b08c971. I didn't see a practical way to automate testing with an actual RViz instance.

Reply:
Thanks!